lib/WebSocket.js

```diff
@@ -214,7 +228,7 @@ class WebSocket extends EventEmitter {
     this.extensions = null;
 
     this.removeAllListeners();
-    this.on('error', function onerror () {}); // catch all errors after this
+    this.on('error', noop); // Catch all errors after this.
```

Comment:
Not related to this PR, but we need start doing a cleaner way of handling this. Right now valid errors might get swallowed without us knowning. Maybe something like `diagnostics`/`debug` would be more useful here so we can at least still log when these errors get swallowed.